src/frequenz/sdk/actor/__init__.py

```diff
@@ -3,12 +3,12 @@
 
 """A base class for creating simple composable actors."""
 
-from . import power_distributing
 from ._channel_registry import ChannelRegistry
 from ._config_managing import ConfigManagingActor
 from ._data_sourcing import ComponentMetricRequest, DataSourcingActor
 from ._decorator import actor
 from ._resampling import ComponentMetricsResamplingActor
+from .power_distributing import power_distributing
```

Comment:
Is this accurate?  This is like not using the `__init__.py` file that you made.

Reply:
Done